packages/local/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "koishi-plugin-booru-local",
+  "description": "Image service by local collection for Koishi",
+  "version": "0.1.0",
```

Comment:
for consistency from other plugins, the version number should start from "1.0.0"